### ISSUE TITLE
Add "VA 508 document audit" instructions to Platform Website

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#27756]
## Issue Description
Ad-hoc 508 audits for PDFs (as well as other documents) are out of scope for Platform accessibility specialists. The VA 508 Office has a document audit process that VFS teams are able to use. We need to ensure that the the guidance for how to request a VA 508 document audit is on Platform Website so we can easily point VFS team members in the right direction.

The instructions are:

```
Steps to request a VA 508 document audit:

1. On the VA Network, click on the YourIT! shortcut. Should open a browser and land on the home page.
2. In the search box, enter "508 audit request" and click on the top result. Top result should be "508 Accessibility Audit Request"
3. You'll get an input form to enter relevant data. At the bottom select "Documents" from the Product Type dropdown
4. Enter Linsey DeLisle as your 508 office contact
5. Upload the document from your machine. (May have to download it from the web first)
6. You should receive a confirmation email with your ticket number linked
7. Lead time is around 1-2 weeks generally, depending on the team's work load
```

---
## Tasks
- [x] Determine where on Platform Website the instructions should live
   - [Testing and releasing / Accessibility testing](https://depo-platform-documentation.scrollhelp.site/test-release/Accessibility-testing.1255931908.html)
- [ ] Create page on Platform Website and add instruction above.

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by shiragoodman** [comment]
hey @JoeTice  - I added your teams labels to this ticket since these instructions will live in the Testing space on Platform Website. 

Maybe one of our shared team members (Trevor, Noah, Dariusz, or Peter) can take on this work. Let me know!
